docs-ref-conceptual/param-persist-howto.md

```diff
@@ -72,15 +72,15 @@ az config param-persist show
 
 ## Persisted parameter and global variable comparison
 
-There are two Azure CLI commands that can be used to default parameter values: `az configure` and `az config param-persist`.  Use the `az configure` command to specify _global variables_ such as group, location, or web.  Use `az param-persist` to specify _local default values_ unique to your workload.  Stored values are used by the CLI in place of required arguments.
+There are two Azure CLI commands that can be used to default parameter values: `az config set defaults` and `az config param-persist`.  Use the `az config set defaults` command to specify _global variables_ such as group, location, or web.  Use `az param-persist` to specify _local default values_ unique to your workload.  Stored values are used by the CLI in place of required arguments.
 
 > [!Important]
 > Persisted parameters override global context values.
 >
 
 | Reference | Scope | Set | Use
 |-|-|-|-|
-[az configure](/cli/azure/reference-index#az_configure) | Scoped globally across the CLI | Set explicitly using `az configure --defaults` | Use for settings such as logging, data collection, and default argument values
+[az config set defaults](/cli/azure/config) | Scoped globally across the CLI | Set explicitly using `az config set defaults` | Use for settings such as logging, data collection, and default argument values
```

Comment:
`default` is a positional argument, not part of the command name.

```suggestion
[az config set defaults.<option>=<value>](/cli/azure/config) | Scoped globally across the CLI | Set explicitly using `az config set defaults.<option>=<value>` | Use for settings such as logging, data collection, and default argument values
```

Actually the functionality of `az config` is a super set of `az config`. The comparison between `az config` and `az config param-persist` is becomes meaningless.